src/actions/CourtReservationActions.js

```diff
@@ -114,7 +108,7 @@ export const onCommerceCourtReservationCreate = ({
         state: null
       })
       .then(() => {
-        sendPushNotification({ title, body, tokens, connection });
+        onCommercePushNotificationSend(notification, commerceId);
```

Comment:
aca no va notificacion, si es el mismo negocio el que esta reservando

Reply:
Esto estaria in progress por el hecho de ver como manejar cuando haya empleados

Reply:
pero que tendria que ver eso?

Reply:
si yo soy un empleado y reservo a otro empleado que no esta en el local en el momento 🥇

Reply:
no se puede eso

Reply:
como que no se puede? si viene un cliente y se quiere atender con juan y yo soy pedro y le reservo a juan desde el comercio